staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -752,7 +752,7 @@ func describePod(pod *corev1.Pod, events *corev1.EventList) (string, error) {
 			w.Write(LEVEL_0, "Node:\t%s\n", pod.Spec.NodeName+"/"+pod.Status.HostIP)
```

Comment:
keeping describe outputting human-readable times in the current timezone seems better to me